packages/fluentui/docs/src/components/ComponentPlayground/componentGenerators.tsx

```diff
@@ -36,6 +37,9 @@ export const Box: KnobComponentGenerators<BoxProps> = {
   children: () => null,
 };
 
+export const Input: KnobComponentGenerators<InputProps> = {
+  children: () => null,
+};
```

Comment:
It should not be required.

Reply:
When removed, I got the following error on viewing examples:
![image](https://user-images.githubusercontent.com/8701120/90510242-e8f68e00-e15a-11ea-8e52-8cc37cd90ea9.png)